### PR TITLE
feat: Align Swift version support and iOS version for the Public SDK (SDKCF-5010)

### PR DESCRIPTION
**Pull request by malayli**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -1,4 +1,6 @@
-// swift-tools-version:5.3
+// swift-tools-version:5.4
```
Comment (imaginaris) [inline Package.swift]: I think you also need to update the template `Package.swift` in RakutenAnalyticsSDK folder in the main repo

**Review by imaginaris — APPROVED** [review]

File: Package.swift (hunk 1 above)
Comment (malayli) [reply Package.swift]: @imaginaris You are right. Done.

**Review by apparition47 — APPROVED** [review]

**Comment by malayli** [comment]
@donnie-jp Is Package.swift updated when there is a new release?
If yes, I'll remove Package.swift modifications in this PR.

**Comment by malayli** [comment]
Actually Package.swift is replaced by each release. I only updated the README file in this PR.

**Pull request merged**